server/src/test/java/com/google/finapp/FinAppIT.java

```diff
@@ -427,4 +429,49 @@ public void createTransactionForAccount_tooLargeAmount_throwsException() throws
         IllegalArgumentException.class,
         () -> spannerDao.createTransactionForAccount(accountId, amount, isCredit));
   }
+
+  @Test
+  public void getRecentTransactionsForAccount_valid() throws Exception {
+    ByteArray fromAccountId = UuidConverter.getBytesFromUuid(UUID.randomUUID());
+    ByteArray toAccountId = UuidConverter.getBytesFromUuid(UUID.randomUUID());
+    BigDecimal fromAccountBalance = new BigDecimal(20);
+    BigDecimal toAccountBalance = new BigDecimal(0);
+    BigDecimal amount = new BigDecimal(10);
+    databaseClient.write(
+        ImmutableList.of(
+            Mutation.newInsertBuilder("Account")
```

Comment:
Can we extract this into a separate method and re-use. 

Also we should use non-UNSPECIFIED enum types.